hm_pyhelper/hardware_definitions.py

```diff
@@ -336,6 +368,8 @@ def is_raspberry_pi() -> bool:
     # Helium Hotspot
     'helium-v1': {
         'FRIENDLY': 'Helium Hotspot',
+        'CPU_ARCH': 'aarch64',
+        'BALENA_DEVICE_TYPE': 'raspberrypi3-64',
```

Comment:
There are both Pi 3 and pi 4 versions of this. Impossible to tell which is which without opening

Reply:
made the values list type to accomodate such cases.

Reply:
Why not go with `armhf` which supports both?